Gemfile

```diff
@@ -0,0 +1,59 @@
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.0"
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```

Comment:
remove all the comment and sort the gem's name  A to Z please

Reply:
done